src/app/static/src/app/store/surveyAndProgram/actions.ts

```diff
@@ -74,25 +74,22 @@ export const actions: ActionTree<SurveyAndProgramDataState, RootState> & SurveyA
             .delete("/disease/survey/")
             .then(() => {
                 commit({type: "SurveyUpdated", payload: null});
-                commit({type: "filteredData/Reset", payload: null}, {root: true});
```

Comment:
Why have these resets been removed? Is it because filteredData updates itself if a file gets deleted, and you actually want to still see the files that remain?

Reply:
This was accidental! The next PR does remove them because moves all resetting logic to the plugin, must have crept into the wrong PR. I'll revert here.